org.feichtmeier.Musicpod.json

```diff
@@ -0,0 +1,230 @@
+{
+     "app-id": "org.feichtmeier.Musicpod",
```

Comment:
feichtmeier.org doesn't seem to resolve

Reply:
This will be blocking the submission

Reply:
okay we have https://ubuntufluttercommunity.org/ as our official homepage
does it matter if the app id in the flatpak does not match the app id in the underlying application?
which is org.feichtmeier.Musicpod
?

Reply:
No you cannot put someone else's domain as your own, check the app id guidelines in the PR checklist.

Reply:
Would my GitHub repo fork work?

com.github.ShadowOfHassen.muspod

Reply:
You aren't submitting the application and no forks won't do. If @Feichtmeier / the project controls https://ubuntufluttercommunity.org/ then you can use that.

>  does it matter if the app id in the flatpak does not match the app id in the underlying application?

It has to match, otherwise dbus bus names will require extra permissions, there will be desktop integration issues, like no window icons on wayland/wrong wmclass etc.

Reply:
OK, thanks, I'll see which @Feichtmeier would be willing to do.